test/testdrive/github-18608.td

```diff
@@ -0,0 +1,72 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
This could probably be SLT too, right?

Reply:
I could make this an SLT instead if you'd prefer.